app/assets/src/components/common/Header.jsx

```diff
@@ -184,7 +184,7 @@ const MainMenu = () => {
         className={cx(cs.item, isSelected("library") && cs.selected)}
         href="/library"
       >
-        My Library
+        My Data
```

Comment:
interesting change... where did this come from? are we still going to refer to it as the user library internally?

Reply:
Joe/Vida complained. Library is used as a term for sample apparently.
I would like to change internally later, once the terms are fixed...